docs/gitops-install.md

```diff
@@ -112,18 +112,15 @@ your shell.
 
 ### Populating the infrastructure
 
-TODO: some examples and documentation on how to build out a cluster
+For an example cluster setup, see our [documentation using Ubuntu and k3s](install-understack-ubuntu-k3s.md).
```

Comment:
This page isn't a good one to reference cause it does cluster creation and also all the stuff in this doc. It needs to go away and the parts we want to keep need to land else where.

Reply:
reverted